Formula/libmarisa.rb

```diff
@@ -1,8 +1,24 @@
 class Libmarisa < Formula
   desc "Static and space-efficient trie data structure"
-  homepage "https://code.google.com/p/marisa-trie/"
-  url "https://marisa-trie.googlecode.com/files/marisa-0.2.4.tar.gz"
-  sha256 "67a7a4f70d3cc7b0a85eb08f10bc3eaf6763419f0c031f278c1f919121729fb3"
+  homepage "https://github.com/s-yata/marisa-trie"
+  revision 1
+
+  stable do
+    url "https://marisa-trie.googlecode.com/files/marisa-0.2.4.tar.gz"
```

Comment:
Also, it might be a good idea to (re-)define 0.2.4 now in terms of the equivalent GitHub repo commit to get rid of the Google Code tarball altogether immediately, but I figured we might as well wait for s-yata/marisa-trie#9 to be merged, request a new tag, and then use that.